src/base-hr721.ts

```diff
@@ -0,0 +1,11 @@
+export abstract class  BaseHR721 {
```

Comment:
Could please add a @see comment that reference the openzeppelin definition?
like this:
```ts
/**
 * @see {@link https://docs.openzeppelin.com/contracts/3.x/api/token/erc721#IERC721}
 */
export abstract class BaseHR721 {...}
```

Reply:
Solved!